apps/web-giddh/src/app/contact/payment-aside/payment-aside.component.scss

```diff
@@ -91,35 +72,177 @@ form label {
+    padding-top: 25px;
+    margin-top: 5px;
+    .account-holder-name{
+        display: flex;
+        font-size: 14px;
+        .due-amount{
+            max-width: 100%;
+            width: 100%;
+        }
+         p{
+             &:first-child{
+                max-width: 450px;
+                width: 100%;
+             }
+             span{
+                 float: right;
+             }
+
+        }
     }
-    :host .aside-pane {
-        max-width: 370px;
+    .bank-deatils-wrapper{
+        border: 1px solid #d9d9d9;
+        border-radius: 2px;
+        margin-top: 3px;
+        display: flex;
+        .readonly-bank-deatils {
+            width: 100%;
+            max-width: 450px;
+            font-size: 14px;
+            background-color: #EAEAEA;
+            padding: 10px;
+            border-right: 1px solid #d9d9d9;
+            .label-value-data{
+                display: flex;
+                align-items: center;
+                margin: 5px 0;
+                label{
+                    margin-right: 5px;
+                }
+                .form-control[disabled], .form-control[readonly], fieldset[disabled] .form-control{
+                    background-color: transparent !important;
+                    display: inline;
+                    width: auto;
+                    height: 22px;
+                    max-width: 90px;
+                    border: none !important;
+                }
+            }
+        }
+        .amount-remark-section{
+            padding: 15px;
+            width: 100%;
+            .form-group{
+                margin-bottom: 0;
+            }
+            .remark-block{
+                width: 100%;
+                textarea{
+                    min-height: 36px;
+                }
+            }
+        }
     }
 }
-@media (max-width: 375px) {
-    :host {
-        max-width: 310px;
+@media (max-width: 1100px) {
+    .payment-top-form {
+        .otp-success-msg {
+            text-align: left;
+            margin: 10px 0;
+        }
     }
-    :host .aside-pane {
-        max-width: 310px;
+    .payment-top-form {
+        display: block;
     }
 }
-@media (max-width: 320px) {
-    :host {
-        max-width: 280px;
+@media (max-width: 767px) {
+    .selected-bank-details
+    .bank-deatils-wrapper
+    .readonly-bank-deatils {
+        max-width: 100% !important;
+    }
+
+    .bank-deatils-wrapper {
+        display: block !important;
+        .readonly-bank-deatils {
+            max-width: 100%;
+        }
     }
-    :host .aside-pane {
-        max-width: 280px;
+    .selected-bank-details {
+        .account-holder-name {
+            p {
+                &:first-child {
+                    max-width: 100%;
+                }
+            }
+        }
     }
 }
+
+
+
+
+@media (max-width: 480px) {
+    .payment-top-form .d-flex .form-group, .payment-top-form .d-flex .form-group:last-child {
+        min-width: 100%;
+    }
+}
+
+// @media (max-width: 768px) {
```

Comment:
You can remove commented code, yes i comment this code when i started working but this task will handover to other so i could not remove this.